src/main/java/core/basesyntax/model/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Bulldozer, You can do it");
```

Comment:
The message 'Common Bulldozer, You can do it' is not very informative. It should indicate that the Bulldozer has started its work. Consider changing it to something like 'Bulldozer is working.'